README.md

```diff
@@ -1,53 +1,68 @@
 # vets-api-clients (VA Lighthouse APIs)
 
-⚠️ **Materials in this repo may be outdated as of October 2019.** This repository contains resources for accessing VA Lighthouse APIs. It includes API specifications, sample clients, and general access information.   Anything **older than April 2019** may be considered deprecated.  **For the most up-to-date Lighthouse API documentation, please visit [developer.va.gov/explore](https://developer.va.gov/explore).**
-
-Welcome to the repository for consumers of **Department of Veterans Affairs (VA) Lighthouse APIs**.  
+Welcome to the repository for consumers of **Department of Veterans Affairs (VA) Lighthouse APIs**.  It includes API specifications, sample clients, and general access information.   Anything **older than April 2019** may be considered deprecated.  **For the most up-to-date Lighthouse API documentation, please visit [developer.va.gov/explore](https://developer.va.gov/explore).**
```

Comment:
Remove 'Anything **older than April 2019** may be considered deprecated.'

Reply:
I think it's important that we provide some date of reference in this README.  It should be apparent from the last modified dates on the files and folders, but unless we are sure we have combed through and removed any deprecated info, people may still consider materials here a source of truth for the Benefits Intake API (fka VBA Document Upload) at least.  That was the reason I edited the README last year to include those dates. I'm open to changing the wording for clarity. cc @darduda

Reply:
I'm thinking we can just say VA Lighthouse APIs in all areas, and that we don't need to use the full term. If they're here, they'll know what VA is. 

Also, I agree with Ryan. We need to have some more solid indication of what is actually deprecated and what is not. Dates and such would be a good way to do this. Otherwise, saying "may be deprecated" leaves room for guesswork and error. Is is possible to archive the deprecated info in another area, or is this too much of an ask for the moment?

Reply:
Thank you @darduda and @ryanlink! Below are my revisions based on your feedback. Pls let me know if you think additional changes are needed!

-----

Welcome to the repository for consumers of **VA Lighthouse APIs**.  It includes API specifications, sample clients, and general access information. This README was last updated December 2020. **For the most up-to-date Lighthouse API documentation, please visit [developer.va.gov/explore](https://developer.va.gov/explore).**

Reply:
Thanks Sophia - can we add "API documentation found in this repo may be outdated." before the last sentence?

Reply:
Welcome to the repository for consumers of VA Lighthouse APIs. It includes API specifications, sample clients, and general access information. This README was last updated December 2020. For the most up-to-date Lighthouse API documentation, please visit developer.va.gov/explore. API documentation found in this repo may be outdated